tokio/src/runtime/basic_scheduler.rs

```diff
@@ -38,15 +39,10 @@ pub(crate) struct BasicScheduler<P: Park> {
     context_guard: Option<EnterGuard>,
```

Comment:
Do we still need this? You seem to be using some functionality to enter the runtime context in the destructor already.

Reply:
I believe those are different contexts. The one used here is the *runtime* context. The context set in the scheduler is setting `Core`.